### PR TITLE
global: legacy code removal

### LINKED ISSUE #14: tests: fix broken imports
- [ ] `from invenio.legacy.bibconvert.xslt_engine import convert`

for more see Travis logs

### PR DESCRIPTION
**Pull request by jalavik**
* INCOMPATIBLE Removes legacy sample workflows, tasks and
  definitions. (closes #14)

Signed-off-by: Jan Aage Lavik <jan.age.lavik@cern.ch>

### CODE REVIEW

**Comment by jirikuncar** [comment]
@jalavik `WorkflowDefinitionError: WorkflowDefinitionError(Workflow 'marcxml_workflow' does not exist, workflow_name: marcxml_workflow, payload: '{}')`

**Comment by jalavik** [comment]
@jirikuncar Yeah, should be good now. :dancer:

File: invenio_workflows/api.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def worker(self):
         try:
             return import_string('invenio_workflows.workers.%s:%s' % (
                 cfg['CFG_BIBWORKFLOW_WORKER'], cfg['CFG_BIBWORKFLOW_WORKER']))
-        except:
+        except:  # noqa
```
Comment (jirikuncar) [inline invenio_workflows/api.py]: please add `Exception` - is there any reason why not?

Comment (jalavik) [inline invenio_workflows/api.py]: I am not sure honestly. I'll add `Exception`, I just did not want to touch more than necessary with the PR. (Although I started moving the imports here due to the sphinx warnings in Travis, but it's everywhere it seems.)

**Pull request merged**